docs/contributing/developing.md

```diff
@@ -345,3 +345,15 @@ pre-commit install
 ```
 
 pre-commit should now run on every commit you make.
+
+
+## Using forks for installation
+
```

Comment:
This could really do with an explanation of why you might want to use a fork, rather than jumping straight into the instructions.

Reply:
Added a bit of context in the interest of moving this forward